community/cypher/docs/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -195,9 +195,9 @@ class QueryPlanTest extends DocumentingTestBase {
       title = "Cartesian Product",
       text =
         """Produces a cross product of the two inputs -- each row coming from the left child will be matched with all the rows from the right child operator.
-          |The following query will join all the people with all the locations and return the cartesian product of the nodes with those labels.
+          |The following query will join all the people with all the teams and return the cartesian product of the nodes with those labels.
```

Comment:
In thinking about better language for the 'cartesian product' docs I updated, I decided to try clarify the difference between the terms 'cross product' and 'cartesian product'. And I find the mathematically, the 'cartesian product' matches what we do correctly, while 'cross product' does not. A cross product of two 3D vectors results in another 3D vector, and not a 3x3 matrix like a cartesian product does. I think the use of 'cross product' in our language has leaked in as a result of the similar term 'cross join' as synonymous for 'cartesian join' in some SQL docs. In fact I read in microsoft docs that a 'cross join' is defined as a 'cartesian product'. Unless someone argues against this, I will start removing instances of 'cross product' terminology from the docs.